File: Java/Beginner/1018/Main.java
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
 
        int moneyInitial = scan.nextInt();
        int money = moneyInitial;
        
        int hundred = money / 100;
            if(hundred > 0) money -= 100*hundred;
        
        int fifty = money/ 50;
            if(fifty > 0) money -= 50*fifty;
        
        int twenty = money/20;
            if(twenty > 0) money -= 20*twenty;
            
        int ten = money/10;
            if(ten > 0) money -= 10*ten;
            
        int five = money/5;
            if(five > 0) money -= 5*five;
            
        int two = money / 2;
            if(two > 0) money -= 2*two;
            
        int one = money;
        
        System.out.printf("%d\n", moneyInitial);
        System.out.printf("%d nota(s) de R$ 100,00\n", hundred);
        System.out.printf("%d nota(s) de R$ 50,00\n", fifty);
        System.out.printf("%d nota(s) de R$ 20,00\n", twenty);
        System.out.printf("%d nota(s) de R$ 10,00\n", ten);
        System.out.printf("%d nota(s) de R$ 5,00\n", five);
        System.out.printf("%d nota(s) de R$ 2,00\n", two);
        System.out.printf("%d nota(s) de R$ 1,00\n", one);

        scan.close();
    }
 
}

File: Cpp/Beginner/1018/main.cpp
#include <iostream>
using namespace std;

int main(void) {

    int input, total, hundred, fifty, twenty, ten, five, two, one;
    cin >> input;
    total = input;

    hundred = total / 100;
        total %= 100;
    fifty = total / 50;
        total %= 50;
    twenty = total / 20;
        total %= 20;
    ten = total / 10;
        total %= 10;
    five = total / 5;
        total %= 5;
    two = total / 2;
        total %= 2;
    one = total;

    cout << input << endl;
    cout << hundred << " nota(s) de R$ 100,00" << endl;
    cout << fifty << " nota(s) de R$ 50,00" << endl;
    cout << twenty << " nota(s) de R$ 20,00" << endl;
    cout << ten << " nota(s) de R$ 10,00" << endl;
    cout << five << " nota(s) de R$ 5,00" << endl;
    cout << two << " nota(s) de R$ 2,00" << endl;
    cout << one << " nota(s) de R$ 1,00" << endl;

}